rollup/eventwatcher/l1client.go

```diff
@@ -0,0 +1,95 @@
+package eventwatcher
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/scroll-tech/go-ethereum"
+	"github.com/scroll-tech/go-ethereum/accounts/abi"
+	"github.com/scroll-tech/go-ethereum/common"
+	"github.com/scroll-tech/go-ethereum/core/types"
+	"github.com/scroll-tech/go-ethereum/log"
+	"github.com/scroll-tech/go-ethereum/rpc"
+
+	"github.com/scroll-tech/go-ethereum/rollup/sync_service"
+)
+
+// L1Client is a wrapper around EthClient that adds
+// methods for conveniently collecting batch events of ScrollChain contract.
+type L1Client struct {
+	ctx                           context.Context
+	client                        sync_service.EthClient
+	confirmations                 rpc.BlockNumber
```

Comment:
```suggestion
```

Reply:
removed in [fbdab78](https://github.com/scroll-tech/go-ethereum/pull/515/commits/fbdab780156d01caeca04dc3f0c8218fe3b9d9d0)